health-sdk/sdk/src/doc/source/event-tracking.rst

```diff
@@ -24,6 +24,25 @@ The ``GiniHealth`` class exposes kotlin flows which you can collect to track eve
    * ``PaymentState.Success(paymentRequest)`` when the payment request is ready and the banking app will be opened.
    * ``PaymentState.Error(throwable)`` when there was an error creating the payment request or opening the banking app.
    * ``PaymentState.Cancel()`` when the user cancelled the payment flow.
+   .. warning::
+       As the SDK is not responsible for navigation flows outside of it, removing the payment fragment from the hierarchy is the responsibility of implementers at ``PaymentState.Success(paymentRequest)`` or ``PaymentState.Cancel()`` events.
+.. code-block:: java
+            launch {
+                    viewModel.openBankState.collect { paymentState ->
```

Comment:
[Suggestion] Maybe it would be clearer if we used `giniHealth.openBankState` ? What do you think?